iceoryx_hoofs/README.md

```diff
@@ -1,7 +1,7 @@
 
-# Eclipse iceoryx utils overview
+# Eclipse iceoryx hoofs overview
 
-The iceoryx utils are our basic building blocks - the foundation of
+The iceoryx hoofs (**H**andy **O**bjects **O**ptimised **F**or **S**afety) are our basic building blocks - the foundation of
```

Comment:
Is this acronym correct? I read a different one in one of the previous files ... don't know which one, there are just too many

Reply:
Jep, see [here](https://github.com/eclipse-iceoryx/iceoryx/issues/482#issuecomment-791323674)

Reply:
can we find an acronym where `H` stands for hypnotoad?

Reply:
I think this would be blasphemy, using Hypnotoad as a helper or a name for building blocks ;). We all praise Hypnotoad and work for it and not the other way around